distro/src/bin/hazelcast

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+PRGDIR="$( cd "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P)"
```

Comment:
Can you maybe add a command what is the purpose of these two lines? To me, it looks like you get the current directory where `hazelcast` script is located and then set up HAZELCASTHOME to one directory upper?!?

Reply:
Yes, that is correct. Added a comment 👍

Reply:
Isn't it just called "parent directory"? :)